tests/modules/nf-core/sourmash/compare/test.yml

```diff
@@ -0,0 +1,9 @@
+- name: sourmash compare test_sourmash_compare
+  command: nextflow run ./tests/modules/nf-core/sourmash/compare -entry test_sourmash_compare -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/sourmash/compare/nextflow.config
+  tags:
+    - sourmash
+    - sourmash/compare
+  files:
+    - path: output/sourmash/test.comp
+    - path: output/sourmash/test.comp.csv
+    - path: output/sourmash/test.comp.labels.txt
```

Comment:
Would be better to have MD5 sums here where possible to ensure that changes to the module produce the same output.

Reply:
oh good call! ok, i'll also update this once the tests are passing, thank you for the suggestion!